### ISSUE TITLE
Loader is not present.

### ISSUE THREAD

**Issue by SinghSatyam** [sampotts/plyr#615]
- [ ] Issue does not already exist
- [ ] Issue observed on https://plyr.io

### Expected behaviour 
Loader should be present when the video is buffering. On slow speed internet connection, a loader/some other way should be present to display that the video is being loaded.

### Actual behaviour
No loader is present. Although I can see it on plyr.io that the progress bar shows that it's loading. But not from the one cloned.
### Environment

- Browser:
Chrome
- Version:  
- Operating System:
Ubuntu 14.04
- Version: 

Players affected:
- [ ] HTML5 Video
- [ ] HTML5 Audio
- [ ] YouTube
- [Yes ] Vimeo

### Steps to reproduce 
-

**Comment by anuluvi** [comment]
I am also looking for that. did you found any hack?

**Comment by sampotts** [comment]
Do you mean loading state for YouTube and Vimeo? HTML5 is working as far as I know unless you're missing some CSS.

**Comment by anuluvi** [comment]
Hey @sampotts I have included `https://cdn.plyr.io/2.0.13/plyr.css` and `https://cdn.plyr.io/2.0.13/plyr.js` in page. but no loading is showing for Vimeo streaming URL.

Instead of this I am doing as a hack like
`players[0].on('seeking', function(event) {

    $(".loader-video").show();
});

players[0].on('seeked', function(event) {

    $(".loader-video").hide();
});`

**Comment by SinghSatyam** [comment]
Yeah. If I am trying to play a vimeo video on a slow network, then the loader should come i.e the video is buffering.
Right now I am getting something like this with a blank black screen.
![image](https://user-images.githubusercontent.com/3114130/28016769-855c0208-6593-11e7-88fd-eaa9c50aed6a.png)


Expected behaviour is something like this:
![image](https://user-images.githubusercontent.com/3114130/28016872-ec07e51c-6593-11e7-9650-f2a9d5f477fa.png)

**Comment by doostinharrell** [comment]
@sampotts / @manish-wedigtech,

I just submitted a PR (https://github.com/sampotts/plyr/pull/1577) to add loading classes when Vimeo and Youtube videos are buffering.  This improves things slightly for @manish-wedigtech. 

For my clients use case I'm planning on writing some style overrides for `.plyr--loading` that will force rendering of the poster via `@extend .plyr--poster;` while rendering a loading icon on top.

Something along those lines may work for you @manish-wedigtech, although you won't see a percentage of your buffering state.

**Comment by beardhatcode** [comment]
@sampotts , this issue can likely be closed :)
